lib/event_relay/destinations/webhook.ex

```diff
@@ -1,28 +1,49 @@
 defmodule ER.Destinations.Webhook do
   require Logger
-  alias ER.Events.Event
 
   def request(
         destination,
-        event
+        event,
+        now \\ DateTime.utc_now()
       ) do
-    body = Event.json_encode!(event)
-    signature = Event.signature(body, signing_secret: destination.signing_secret)
     url = destination.config["endpoint_url"]
 
+    payload = to_payload(event, now)
+
+    unix_timestamp = DateTime.to_unix(now)
+
+    signature =
+      Webhoox.Authentication.StandardWebhook.sign(
+        event.id,
+        unix_timestamp,
+        payload,
+        destination.signing_secret
+      )
+
     Req.post(
       url: url,
-      body: body,
+      body: Jason.encode!(payload),
       headers: [
         content_type: "application/json",
-        x_event_relay_signature: signature,
+        webhook_id: event.id,
+        webhook_timestamp: unix_timestamp,
+        webhook_signature: signature,
+        x_event_relay_event_id: event.id,
```

Comment:
I wonder if this is to much duplication (webhook_id and x_event_relay_event_id being the same). Maybe they should be different identifiers?